src/easy_config/__init__.py

```diff
@@ -165,6 +171,21 @@ def load(
         if _additional_files:
             for f in _additional_files:
                 values.update(cls._load_file(f))
+        if _lookup_config_envvar is not None:
+            envvar = f'{cls.NAME.upper()}_{_lookup_config_envvar.upper()}'
+
+            try:
+                f = os.environ[envvar]
+            except KeyError as e:
+                raise KeyError(f'No value set for {envvar} in environment') from e
+
+            if not f:
+                raise ValueError(f'Empty value set for {envvar} in environment')
```

Comment:
These two should definitely not be errors but should just be ignored.

Reply:
(Like, given the situation that a program wants to allow parsing a config file passed in the environment but doesn't require it and doesn't want to use a cli option to turn that on or off.)

Reply:
Your choice. Maybe you can go ahead and remove the parts you don't want

Reply:
#10